.github/workflows/test.yaml

```
@@ -113,7 +113,7 @@ jobs:
           # by GitHub. The availability of the Python version
           # depends on the version of Ubuntu in use. 
           # See https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
-          - python-version: '3.8.12'
+          - python-version: '3.9'
```

If 3.9.0 doesn't work... we should see which oldest micro release works.

With `3.9` the tests will run with latest micro-release... but here we want to explictly run with the oldes micro-release


maybe the comment above should be improve to make it clear what's the scope of this test

```suggestion
          - python-version: '3.9.1'
```

---

3.9.0 isn't available for Ubuntu 22.04.  It looks like 3.9.12 is the oldest version of 3.9 that's available for Ubuntu 22.04, so I guess that's what we should use.